fx-src/FirefoxStorage.js

```diff
@@ -299,6 +302,40 @@ export default class FirefoxAdapter extends BaseAdapter {
         return result[0].getResultByName("last_modified");
       });
   }
+
+  /**
+   * Reset the sync status of every record and collection we have
+   * access to.
+   */
+  resetSyncStatus() {
```

Comment:
How is it supposed to be used if it only exists on a specific adapter? Don't we want to expose a method at the root object?